### DISCUSSION TITLE
[MacOS][Sonoma][SSL] warden.test certificate has expired

### DISCUSSION THREAD

**Discussion by eugene-petrov** [wardenenv/warden#715] [Help & Support] [answered]
<img width="470" alt="image" src="https://github.com/wardenenv/warden/assets/7125242/7860a52c-ec6d-4c3e-91d8-c18133cd737c">

<img width="487" alt="image" src="https://github.com/wardenenv/warden/assets/7125242/ed4b04bb-5442-4970-9444-82243da5d548">

**Comment by bap14** [comment, marked as answer]
This is local to your machine, not a global item. You should be able to resolve this by running `warden sign-certificate warden.test`

**Reply by eugene-petrov** [reply]
Hello @bap14! Thank you for your support!
I have updated my Mac and have moved from den to warden.
So yes, it could be something with my local environment.

I have tried 
- warden sign-certificate warden.test command
- removing the certificate and running warden install
- removing warden and installing warden again
- removing den via brew, reloading my computer, running brew doctor and so on

Is there anything I might try I have forgotten? Thank you!

**Reply by eugene-petrov** [reply]
Actually, after executing the warden sign-certificate warden.test command and reloading my browser it finally worked out.
I tried on different instances like warden sign-certificate some-magento2.test, but I should also have executed warden sign-certificate warden.test.
Thank you again for your helpful advice.
